mungegithub/Dockerfile-shame-mailer

```diff
@@ -12,36 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The following environment variable need to be bound before executing
-# this containter:
-#
-#   GITHUB_API_TOKEN: An API token to use with Github's API.
```

Comment:
Keep this documentation of what the precise values are? I don't see it anywhere else. I think the next commit means that this should work as normal, anyways?